app/index.html

```diff
@@ -33,37 +33,243 @@
     -->
 
     <!-- Page styles -->
+    <link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.0.2/material.indigo-pink.min.css">
```

Comment:
We may want to add some brief documentation here explaining how to choose your own colours.